tasks/run-matlab-tests/v1/task.json

```diff
@@ -13,59 +13,31 @@
         "Patch": 0
     },
     "groups": [
-        {
-            "name": "testArtifacts",
-            "displayName": "Generate Test Artifacts"
-        },
-        {
-            "name": "coverageArtifacts",
-            "displayName": "Generate Coverage Artifacts"
-        },
         {
           "name": "filterTests",
           "displayName": "Filter Tests"
         },
         {
           "name": "customizeTestRun",
           "displayName": "Customize Test Run"
-        }
-    ],
-    "inputs": [
-        {
-            "name": "codeCoverageCobertura",
-            "type": "string",
-            "label": "Cobertura code coverage",
-            "defaultValue": "",
-            "required": false,
-            "groupName": "coverageArtifacts",
-            "helpMarkDown": "Path to write code coverage report in Cobertura XML format."
         },
         {
-            "name": "modelCoverageCobertura",
-            "type": "string",
-            "label": "Cobertura model coverage",
-            "defaultValue": "",
-            "groupName": "coverageArtifacts",
-            "required": false,
-            "helpMarkDown": "Path to write model coverage report in Cobertura XML format (requires Simulink Coverage™ license and is supported in MATLAB R2018b or later)."
+            "name": "testArtifacts",
+            "displayName": "Generate Test Artifacts"
         },
         {
-            "name": "testResultsSimulinkTest",
-            "type": "string",
-            "label": "Simulink test results",
-            "defaultValue": "",
-            "groupName": "testArtifacts",
-            "required": false,
-            "helpMarkDown": "Path to export Simulink Test Manager results in MLDATX format (requires Simulink Test license and is supported in MATLAB R2019a or later)."
-        },
+            "name": "coverageArtifacts",
+            "displayName": "Generate Coverage Artifacts"
+        }
+    ],
+    "inputs": [
         {
-            "name": "testResultsPDF",
+            "name": "sourceFolder",
             "type": "string",
-            "label": "PDF test report ",
+            "label": "Source folder",
             "defaultValue": "",
-            "groupName": "testArtifacts",
             "required": false,
-            "helpMarkDown": "Path to write test results report in PDF format (requires MATLAB R2020b or later on macOS platforms)."
+            "helpMarkDown": "Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list."
```

Comment:
Can we shorten by removing this part?

The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. 

I am hoping the user refers to the doc for more info.